modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/criteria_builder/TypedInput.es.js

```diff
@@ -46,75 +44,59 @@ class TypedInput extends React.Component {
 	}
 
 	_handleChange = event => {
-		const {onChange} = this.props;
-
-		const value = event.target.value;
-
-		onChange(value);
+		this.props.onChange(event.target.value);
```

Comment:
@thektan I really like this one-liners. I'm guessing I can use them when the alternative is destructuring a single value. Is that correct?

Reply:
@andresfulla Yup! No need to destructure or define new variables if it's only used once or if there's only 1 value that's being destructured.